recipes-tag/12-id-smi-analysis/meta.yaml

```diff
@@ -0,0 +1,14 @@
+{% set year = "2018" %}
+{% set cycle = "2" %}
+{% set version ="0" %}
+
+package:
+  name: 12-id-smi-analysis
+  version: {{ year }}-{{ cycle }}.{{ version }}
```

Comment:
conda doesn't allow "-" in version number. Can you correct this, we use C for other pkgs. https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/07-bm-qas-collection/meta.yaml 



Error: bad character '-' in package/version: 2018-2.0
 raised with check_output comand ['conda', 'build', '/repo/recipes-tag/12-id-smi-analysis', '--output', '--python', '3.5', '--numpy', '1.11']
['conda', 'build', '/repo/recipes-tag/12-id-smi-collection', '--output', '--python', '3.5', '--numpy', '1.11']
Error: bad character '-' in package/version: 2018-2.0